src/backend/tests/Fusion.Resources.Api.Tests/IntegrationTests/PersonAbsenceTests.cs

```diff
@@ -98,6 +101,110 @@ public async Task DeleteAbsence_ShouldBeOk_WhenAdmin()
             response.Should().BeSuccessfull();
         }
 
+        [Fact]
+        public async Task ShouldGetRoleNameFromBasePositionWhenEmpty()
```

Comment:
Should try and align test names with other?

`[GeneralOperation]_Should[Something]_When[condition]`

I Guess here it would be `CreateAbsence_ShouldUseBasePositionName_WhenRoleNameNull` ?